src/bin/ion/commands/inspect.rs

```diff
@@ -825,6 +831,16 @@ impl<'a, 'b> IonInspector<'a, 'b> {
             Ok(())
         })?;
 
+        // TODO: This does not account for shared symbol table imports. However, the CLI does not
+        //       yet support specifying a catalog, so it's correct enough for the moment.
+        let symtab_value = symtab_struct.as_value();
+        let mut next_symbol_id = symtab_value.symbol_table().len();
```

Comment:
🗺️ We don't have to track which ID we were on because we can ask the symbol table what the next ID will be.